### PR TITLE
add diagnotic s01e03

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds a new episode diagnostic s01e03 which will be sent out as an sms ad with catch-all trigger but won't be aired as a radio show. Its technically an activation but not really a rqa question. 

Have also renamed previous diagnostic episodes pipeline variables where possible to standardize them.

### CODE REVIEW

File: configuration/code_schemes.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class CodeSchemes(object):
     S01E10 = _open_scheme("s01e10.json")
     S01E11 = _open_scheme("s01e11.json")
     S01E12 = _open_scheme("s01e12.json")
+    DIAGNOSTIC_S01E03 = _open_scheme("diagnostic_s01e03.json")
```
Comment (as2388) [inline configuration/code_schemes.py]: I wonder if it makes for this line to be next to the other diagnostics. It depends whether this is more useful to be organised by type or by time.

File: configuration/pipeline_config.json [hunk 2]
```diff
@@ -7,20 +7,7 @@
...
-        "imaqal_covid19_mag_s01e03_activation",
-        "imaqal_covid19_mag_s01e04_activation",
-        "imaqal_covid19_mag_s01e05_activation",
-        "imaqal_covid19_mag_s01e06_activation",
-        "imaqal_covid19_mag_s01e07_activation",
-        "imaqal_covid19_mag_s01e08_activation",
-        "imaqal_covid19_mag_s01e09_activation",
-        "imaqal_covid19_mag_s01e10_activation",
-        "imaqal_covid19_mag_s01e11_activation",
-        "imaqal_covid19_mag_s01e12_activation"
```
Comment (as2388) [inline configuration/pipeline_config.json:23]: Why are we removing all the old shows? Was this accidentally committed from testing?

**Review by as2388 — APPROVED** [review]

File: configuration/code_schemes.py (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/code_schemes.py]: by type is better - updated thanks

File: configuration/pipeline_config.json (hunk 2 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json:23]: accidentally committed thanks for catching this

**Pull request merged**